src/Identity/ApiAuthorization.IdentityServer/samples/ApiAuthSample/Pages/Index.cshtml

```diff
@@ -9,6 +9,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <base href="~/" />
```

Comment:
Just checking: Does the tag helper stuff properly resolve a `<base href="...">` element like other link elements? I've never tried that so I'm not sure.

Reply:
Yeah, our blazor templates do it this way too.